### ISSUE TITLE
INFO: Is there a way to add ShellCheck to tox?

### ISSUE THREAD

**Issue by srisankethu** [koalaman/shellcheck#802]
Is there a way to add shellcheck to tox(without using `whitelist_externals`)?

**Comment by austin987** [comment]
I thought tox was for python only..?

**Comment by srisankethu** [comment]
Yes indeed! But there is `nodeenv` that allows us to use nodejs, npm, node by creating a python env. I wanted to know if there is any such *trick* to use shellcheck in tox.

**Comment by lokal-profil** [comment]
[shellcheck-py](https://pypi.org/project/shellcheck-py/) should work on python > 3.5
